docs/Files/UI/Solution-Manager/Library/MasterJobs/Add-Jobs.md

```diff
@@ -0,0 +1,40 @@
+# Adding Jobs
+
+To add a Master Job, go to **Library** > **Master Jobs** and select **Add**.
+
+The Master Job Details page is displayed:
+
+![Master Job Details](master-job-details.png 'Master Job Details')
+
+Select a Schedule, enter a Job Name. Select a Job Type, Department, and Access Code.
```

Comment:
I would break this out into steps, or have each input be a new line (like you do for the others below)